src/Adapter/MSTest.TestAdapter/Execution/TypeCache.cs

```diff
@@ -390,7 +390,7 @@ private TestAssemblyInfo GetAssemblyInfo(Type type)
 
         assemblyInfo = new TestAssemblyInfo(assembly);
 
-        Type[] types = AssemblyEnumerator.GetTypes(assembly, assembly.FullName!, null);
```

Comment:
Removing this should produce a warning. `Assembly.FullName` is annotated as nullable on .NET Core.